assets/src/models/vehicleStatus.ts

```diff
@@ -0,0 +1,28 @@
+import { ScheduleAdherenceStatus, Vehicle } from "../skate"
+
+export type VehicleAdherenceStatus = ScheduleAdherenceStatus | "off-course"
+
+export const status = (vehicle: Vehicle): VehicleAdherenceStatus =>
+  isUnassignedBySwiftly(vehicle)
+    ? "off-course"
+    : vehicle.scheduleAdherenceStatus
+
+export const isUnassignedBySwiftly = ({
```

Comment:
I think one reason I wanted to combine `VehicleAdherenceStatus` and `ScheduleAdherenceStatus` is so a bunch of files don't need to know details about Swiftly. Another way to accomplish the same goal could be to rename this function to (or export a different function) `isOffCourse` instead of `isUnassignedBySwiftly`, so the callers can think about the meaning rather than the implementation.

Reply:
I like that 👍